geowebcache/azureblob/src/main/java/org/geowebcache/azure/AzureBlobStoreInfo.java

```diff
@@ -41,7 +41,7 @@
      */
     public static final int DEFAULT_CONNECTIONS = 100;
 
-    static Log log = LogFactory.getLog(AzureBlobStoreInfo.class);
+    private static final Logger LOGGER = Logger.getLogger(AzureBlobStoreInfo.class.getName());
```

Comment:
GWC uniformly uses commons logging, please revert this change.

Reply:
Thanks, reverted.